HerPublicWebsite/Controllers/ReferralRequestFollowUpController.cs

```diff
@@ -0,0 +1,58 @@
+using System.Threading.Tasks;
+using HerPublicWebsite.Models.ReferralRequestFollowUp;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Routing;
+using HerPublicWebsite.BusinessLogic.Services.ReferralFollowUps;
+using HerPublicWebsite.Models.Enums;
+using HerPublicWebsite.BusinessLogic.Models;
+
+namespace HerPublicWebsite.Controllers;
+
+[Route("referral-follow-up")]
```

Comment:
Might be a better name for all the route components in this file. Open to suggestions!

Reply:
Seems reasonable as a base route to me